netlify.toml

```
@@ -32,4 +32,4 @@
 [[headers]]
   for = "/*"
   [headers.values]
-    Content-Security-Policy = "script-src 'self' *.googletagmanager.com *.cloudfront.net js.hs-scripts.com;"
+    Content-Security-Policy = "script-src 'self' *.googletagmanager.com *.cloudfront.net js.hs-scripts.com js.hsadspixel.net js.hsleadflows.net *.openzeppelin.com *.netlify.app js.hs-analytics.net js.hubspot.com js.headleadflows.net js.hscollectedforms.net js.hs-banner.com;"
```

Removing `*.netlify.app` because anyone can deploy something at that domain, and specifying `wizard.openzeppelin.com` instead of a wildcard for `openzeppelin.com` just in case.

```suggestion
    Content-Security-Policy = "script-src 'self' *.googletagmanager.com *.cloudfront.net js.hs-scripts.com js.hsadspixel.net js.hsleadflows.net wizard.openzeppelin.com js.hs-analytics.net js.hubspot.com js.headleadflows.net js.hscollectedforms.net js.hs-banner.com;"
```

---

I'm actually thinking `*.cloudfront.net` might be the same? If anyone can deploy a script at that domain then the CSP is not doing anything, am I thinking about this correctly?

---

I believe the correct thing would be to use the specific CloudFront subdomain that Netlify uses to serve assets. As far as I can tell this is available in the environment variable `NETLIFY_IMAGES_CDN_DOMAIN` ([source](https://docs.netlify.com/configure-builds/environment-variables/)). However, environment variables can't be directly used in `netlify.toml` so we would have to write a script to create a [`_headers` file](https://docs.netlify.com/routing/headers/#syntax-for-the-headers-file) using the value of this variable.

---

Thanks for the fixes, @frangio. Currently, the specific domain being used is d33wubrfki0l68.cloudfront.net, but if we specify it and it eventually changes, it would break the whole website. We could have it as a wildcard for now while we think of a better solution - even though I agree that it wouldn't be as secure since attackers could still leverage cloudfront.net. We would reduce the attack surface pero no mucho.